ui/src/templates/components/CommunityTemplateListGroup.tsx

```diff
@@ -16,7 +16,7 @@ import {
 
 interface Props {
   title: string
-  count?: string
+  count: number
```

Comment:
i assume the default value of numbers in ts is 0?

Reply:
@glinton good question - `count` actually comes from https://github.com/influxdata/influxdb/blob/master/ui/src/templates/components/CommunityTemplateContents.tsx#L44 - each collection has `.length` called on it. before, it would crash because it was calling `.length` on `null`. the tricksy change here is to make all the collections default to empty arrays so `.length` always returns 0.